cmsdb/campaigns/run2_2016_nano_uhh_v12/ewk.py

```diff
@@ -0,0 +1,746 @@
+# coding: utf-8
+
+"""
+Electroweak datasets for the 2016 data-taking campaign with datasets at NanoAOD tier in
+version 12, created with custom content at UHH.
+"""
+
+import cmsdb.processes as procs
+from cmsdb.campaigns.run2_2016_nano_uhh_v12 import campaign_run2_2016_nano_uhh_v12 as cpn
+
+
+#
+# Drell-Yan
+#
+cpn.add_dataset(
+    name="dy_lep_m50_amcatnlo",
+    id=14284445,
+    processes=[procs.dy_lep_m50],
+    keys=[
+        "/DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIISummer20UL16NanoAODv12-106X_mcRun2_asymptotic_v17-v1/NANOAODSIM",  # noqa
+    ],
+    n_files=41,
+    n_events=73493454,
+)
+
+# jet binned, madgraph
```

Comment:
Are the commented datasets still being processed? If not, we can also remove them here.

Reply:
no, we only process the amcatnlo ones for the jet binning and don't use the  HT binned but pt binned ones. I removed the ones we did not process.